src/HackF5.UnitySpy/Detail/MemoryObject.cs

```diff
@@ -14,11 +14,6 @@ protected MemoryObject(AssemblyImage image, IntPtr address)
             this.Address = address;
         }
 
-        public IntPtr GetAddress()
```

Comment:
Not really sure why this was here. Maybe at one point the IMemoryObject interface had this method declaration?
Edit: spelling